grails-app/i18n/messages_ca_ES.properties

```diff
@@ -1040,6 +1040,7 @@ kuorum.web.commands.editor.EditorAccountCommand.alias.nullable = L’àlies no p
 kuorum.web.commands.editor.EditorAccountCommand.alias.unique = Aquest àlies ja s’està fent servir
 kuorum.web.commands.editor.EditorAccountCommand.bio.label = Description
 kuorum.web.commands.editor.EditorAccountCommand.bio.placeHolder= Explica'ns més sobre tu mateix o la teva organització
+kuorum.web.commands.editor.EditorAccountCommand.bio.maxCharsHtml.error.kuorum.web.commands.editor.EditorAccountCommand.bio = Màxim {3} caràcters amb espais
```

Comment:
Same. The code is to long.
kuorum.web.commands.editor.EditorAccountCommand.bio.maxCharsHtml